build.gradle

```diff
@@ -17,8 +17,7 @@ buildscript {
 
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
-        classpath 'org.terasology:reflections:0.9.12-MB' // This goes away in v8
+        classpath 'com.android.tools.build:gradle:7.4.1'
```

Comment:
```suggestion
        classpath 'org.terasology:reflections:0.9.12-MB' // This goes away in v8
        classpath 'com.android.tools.build:gradle:7.4.1'
```

Could you please add the reflections dependency back into the build script dependencies? The Android parts of gestalt thows gradle errors without it:
```
FAILURE: Build failed with an exception.

* Where:
Build file 'gestalt\gestalt-android-testbed\build.gradle' line: 4

* What went wrong:
Could not compile build file 'gestalt\gestalt-android-testbed\build.gradle'.
> startup failed:
  build file 'gestalt\gestalt-android-testbed\build.gradle': 4: unable to resolve class org.reflections.Reflections
   @ line 4, column 1.
     import org.reflections.Reflections
     ^

  build file 'gestalt\gestalt-android-testbed\build.gradle': 5: unable to resolve class org.reflections.scanners.ResourcesScanner
   @ line 5, column 1.
     import org.reflections.scanners.ResourcesScanner
     ^

  build file 'gestalt\gestalt-android-testbed\build.gradle': 6: unable to resolve class org.reflections.scanners.SubTypesScanner
   @ line 6, column 1.
     import org.reflections.scanners.SubTypesScanner
     ^

  build file 'gestalt\gestalt-android-testbed\build.gradle': 7: unable to resolve class org.reflections.scanners.TypeAnnotationsScanner
   @ line 7, column 1.
     import org.reflections.scanners.TypeAnnotationsScanner
     ^

  build file 'gestalt\gestalt-android-testbed\build.gradle': 8: unable to resolve class org.reflections.serializers.JsonSerializer
   @ line 8, column 1.
     import org.reflections.serializers.JsonSerializer
     ^

  build file 'gestalt\gestalt-android-testbed\build.gradle': 9: unable to resolve class org.reflections.util.ConfigurationBuilder
   @ line 9, column 1.
     import org.reflections.util.ConfigurationBuilder
     ^

  build file 'gestalt\gestalt-android-testbed\build.gradle': 80: unable to resolve class org.reflections.Configuration
   @ line 80, column 35.
         org.reflections.Configuration config = new ConfigurationBuilder()
```